File: generators/service-naturalLanguageUnderstanding/templates/python/instrumentation.py
from ibmcloudenv import IBMCloudEnv
from watson_developer_cloud import NaturalLanguageUnderstandingV1

natural_language_understanding = NaturalLanguageUnderstandingV1(
    username=IBMCloudEnv.getString('naturalLanguageUnderstanding_username'),
    password=IBMCloudEnv.getString('naturalLanguageUnderstanding_password'),
    version='2017-02-27')

def getService(app):
    return 'watson-natural-language-understanding', natural_language_understanding

File: generators/service-naturalLanguageUnderstanding/templates/node/instrumentation.js
var IBMCloudEnv = require('ibm-cloud-env');
var NaturalLanguageUnderstandingV1 = require('watson-developer-cloud/natural-language-understanding/v1.js');

module.exports = function(app, serviceManager){
    var naturalLanguageUnderstanding = new NaturalLanguageUnderstandingV1({
	    url: IBMCloudEnv.getString('naturalLanguageUnderstanding_url'),
	    username: IBMCloudEnv.getString('naturalLanguageUnderstanding_username'),
        password: IBMCloudEnv.getString('naturalLanguageUnderstanding_password'),
        version_date: NaturalLanguageUnderstandingV1.VERSION_DATE_2017_02_27
    });
    serviceManager.set("watson-natural-language-understanding", naturalLanguageUnderstanding);
};